server/api/include/irods/rs_get_resource_info_for_operation.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef RS_GET_RESOURCE_INFO_FOR_OPERATION_HPP
+#define RS_GET_RESOURCE_INFO_FOR_OPERATION_HPP
+
+#include "irods/rcConnect.h"
+#include "irods/dataObjClose.h"
```

Comment:
This should be `dataObjInpOut.h`.

Reply:
replacing with struct DataObjInp fwd-ref